integrated_channels/integrated_channel/models.py

```diff
@@ -296,6 +297,7 @@ class ContentMetadataItemTransmission(TimeStampedModel):
         blank=True,
         null=True
     )
+    enterprise_customer_catalog_uuid = models.UUIDField()
```

Comment:
You'll need to commit a migration file for this and also it probably should be set as nullable/blank.

Reply:
Also not sure if we should consider referential integrity here and specify the UUID as an FK to the ECC table.

Reply:
Yup- migration is in the works now. The trick is that I want to backfill all previous metadata transmission audits with their specific catalog's uuid. This way the field can be not nullable or blank and we won't have to worry about recovering from our present state.

Reply:
I'm not sure if we can FK here since the tables are across db's (ie platform vs ECC)